README.md

```diff
@@ -7,33 +7,59 @@ https://storage.googleapis.com/download.tensorflow.org/models/tflite_11_05_08/re
 
 To validate TensorFlow model on one image run:
 ```bash
-python demo_classification_tf.py --graph /path/to/resnet_v2_101_299_frozen.pb --image /path/to/example.jpeg
+python3 demo_classification.py --graph /path/to/resnet_v2_101_299_frozen.pb --image /path/to/example.jpeg --engine tf
```

Comment:
please avoid extra information in the example command line. It's hard for users to modify it. Just
```
python3 demo_classification.py --graph resnet_v2_101_299_frozen.pb --engine tf --image example.jpeg
```